### PR TITLE
(0474.035-072) Converted A good chunk of Cicero

### PR DESCRIPTION
**Pull request by PonteIneptique**
Not all is passing but at least that's a good chunk :)

### CODE REVIEW

**Comment by sonofmun** [comment]
I wish we had the new Hook UI so I could easily tell what was passing before.

File: data/phi0474/phi035/phi0474.phi035.perseus-eng1.xml [hunk 1]
```diff
@@ -1,118 +1,113 @@
...
+         </refsDecl>
+         <refsDecl n="TEI.2">
+            <refState delim="." unit="speech"/>
+            <refState delim="." unit="chapter"/>
+            <refState unit="section"/>
+         </refsDecl>
+      </encodingDesc>
+      <profileDesc >
+         <langUsage>
+            <language ident="">English</language>
```
Comment (sonofmun) [inline data/phi0474/phi035/phi0474.phi035.perseus-eng1.xml]: You should have the language codes here.

File: data/phi0474/phi035/phi0474.phi035.perseus-eng1.xml [hunk 2]
```diff
@@ -1,118 +1,113 @@
...
-				<item> 
-$Log: cic.yonge4_eng.xml,v $
-Revision 1.3  2010-08-31 18:43:05  rsingh04
-cleaned up log keyword
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/8.22/tei-epidoc.rng"
+  schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="http://www.stoa.org/epidoc/schema/8.22/tei-epidoc.rng"
+  schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+   <teiHeader type="text">
```
Comment (sonofmun) [inline data/phi0474/phi035/phi0474.phi035.perseus-eng1.xml]: Add `xml:lang="eng"` to the teiHeader element.

File: data/phi0474/phi037/__cts__.xml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:latinLit:phi0474.phi037" xml:lang="lat" groupUrn="urn:cts:latinLit:phi0474">
+            <ti:title xml:lang="lat">On Oratory</ti:title>
```
Comment (sonofmun) [inline data/phi0474/phi037/__cts__.xml]: lang tag should be "eng"

File: data/phi0474/phi037/__cts__.xml [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+            <ti:edition urn="urn:cts:latinLit:phi0474.phi037.perseus-lat1" workUrn="urn:cts:latinLit:phi0474.phi037">
+              <ti:label xml:lang="eng">On Oratory, Rhetorica Tomus I</ti:label>
```
Comment (sonofmun) [inline data/phi0474/phi037/__cts__.xml]: lang tag should be "mul"

File: data/phi0474/phi036/__cts__.xml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+<ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:latinLit:phi0474.phi036" xml:lang="lat" groupUrn="urn:cts:latinLit:phi0474">
+            <ti:title xml:lang="lat">De Inventione</ti:title>
+            <ti:edition urn="urn:cts:latinLit:phi0474.phi036.perseus-lat1" workUrn="urn:cts:latinLit:phi0474.phi036">
+              <ti:label xml:lang="eng">De Inventione, Rhetorici libri duo qui vocantur de inventione</ti:label>
```
Comment (sonofmun) [inline data/phi0474/phi036/__cts__.xml]: lang tag should be "lat"

File: data/phi0474/phi036/phi0474.phi036.perseus-lat1.xml [hunk 6]
```diff
@@ -1,55 +1,75 @@
...
-      <sourceDesc>
-        <biblStruct>
-          <monogr>
-            <title>Rhetorici libri duo qui vocantur de inventione</title>
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/8.22/tei-epidoc.rng"
+  schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="http://www.stoa.org/epidoc/schema/8.22/tei-epidoc.rng"
+  schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+   <teiHeader type="text">
```
Comment (sonofmun) [inline data/phi0474/phi036/phi0474.phi036.perseus-lat1.xml]: Add `xml:lang="eng"` to teiHeader.

**Comment by sonofmun** [comment]
Mostly it looks good. The things that need to be changed are the `xml:lang` attributes. Basically, always have `xml:lang="eng"` on the teiHeader element. Then add the appropriate language tag to any elements that are not English. And be careful with the language tags in the metadata files. I commented on some things. Hopefully you can see the comments on the files involved in the PR.

**Comment by lcerrato** [comment]
Ok, these files are a good start but be aware that I will have more work to do.
We are bumping the names, adding to the CITE tables, and removing old files and tracking files whenever a CTS EpiDoc conversion takes place. I'm also checking things like the headers and publication dates here as most of the P4 files have problems in those areas, and fixing quotes and entities, and for Cicero, the canonical structure was a problem. Most of these works are using the wrong sections vs. milestones so I spent some time converting to the OLD schemes.
As I have some of this work in another branch, I can review this and perform those steps but would prefer to work on them in smaller (!) issue batches.

**Comment by PonteIneptique** [comment]
Sure @lcerrato 
The issue being I am leaving soon and will need the whole corpus to work. If needed, I can keep the modifications for myself ?
Best

**Comment by lcerrato** [comment]
I understand.  I am doing my best to maintain a consistent workflow and meet the needs of the entire Perseus community as the frontline support person. There are some established guidelines we have been trying to follow to keep things sane and tractable.
Cicero was what I had prioritized for student work as I have already reviewed many of the issues in the earlier batch of work and have some files here. 
If you want to commit things before you leave and don't want to wait for me to weigh on on PR then I would ask that you create work-based issues (not large batches) so that we have placeholders to tell us to go back and review these works for things like URN bumping, canonical structure changes and bibliographic accuracy. 
Perhaps just script a bunch of generic issues for all of the works you commit that will enable me to track what the changes were? Thanks.

**Comment by PonteIneptique** [comment]
I know you're doing your best and in no case was I trying to be rude, and hope it was not seen as such.
I'll happily make issue when needed about what is required to still do :)

**Pull request merged**